daemon/daemon.go

```diff
@@ -1013,8 +1013,8 @@ func (daemon *Daemon) changes(container *container.Container) ([]archive.Change,
 	return container.RWLayer.Changes()
 }
 
-// TagImage creates a tag in the repository reponame, pointing to the image named
-// imageName.
+// TagImage creates a tag in the repository newTag.Name(), pointing to the
```

Comment:
I think the "and" is extraneous. Otherwise, it definitely looks like an improvement.